### PR TITLE
Add lama.txt

### PR DESCRIPTION
**Pull request by lamacoding**

### CODE REVIEW

File: lama.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Test file
```
Comment (Amadeus1791) [inline lama.txt]: I don't like this text. Please make the text a little longer.

Comment (lamacoding) [reply lama.txt]: Nope

Comment (Mario7400) [reply lama.txt]: hi

Comment (Mario7400) [reply lama.txt]: :)

**Pull request merged**